docs/dealing-with-large-issue-metrics.md

```diff
@@ -0,0 +1,17 @@
+# Dealing with large issue metrics Markdown files
+
+When working with lots of issues/pull requests/discussion results, the resulting issue_metrics.md  file can become very large. This can cause the GitHub API to return an error when trying to create an issue with the contents of the file.
+
+```shell
+Pull request creation failed. Validation failed: Body is too long (maximum is 65536 characters)
+```
+
+To work around this limitation, the issue-metrics action detects the issue and splits the issue_metrics.md file into smaller files. So instead of issue_metrics.md, you will get issue_metrics_0.md, issue_metrics_1.md, etc.
```

Comment:
```suggestion
To work around this limitation, the issue-metrics action detects the large file size and splits the issue_metrics.md file into smaller files. So instead of issue_metrics.md, you will get issue_metrics_0.md, issue_metrics_1.md, etc.
```